src/PageContent/RequestPickup/RecurringPickupRequest.js

```diff
@@ -0,0 +1,267 @@
+import React, {Component} from 'react';
+import firebase from '../../FirebaseConfig.js';
+import { RequestRepeatType, RequestDurationType } from '../../Enums.js';
+import './RequestPickup.css';
+import PickupSummary from './PickupSummary.js';
+
+class RecurringPickupRequest extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            memberList: [
+                {id: 'dhA03LwTp3cibXVUcb3nQqO34wj1', name: 'Test DA1 Member1'}, 
+                {id: 'fbCm3Yrbi4e12WgpVz3gq25VKea2', name: 'Test DA1 Member2'}
+            ],
+            delivererGroups: [
+                {id: 'R8BAHrxdkfQoAmfWvGa1OJmjQP43', name: 'Test DG1'},
+                {id: 'sS4dqgxgLIXtPf42DydgkWLWeHT2', name: 'Test DG2'}
+            ],
+            receivingAgencies: [
+                {id: 'uCm0OG4WeoSyjk3c0rdY3mlaBXl2', name: 'Test RA2'},
+                {id: 'uGOFJ8NqHjbZhKAYzSZFRs1dSKD3', name: 'Test RA1'}
+            ],
+            fields: {},
+            errors: {},
+            isOpen: false,
+            formInfo: []
+        }; 
+        this.submitRequest = this.submitRequest.bind(this);
+        this.createRequest = this.createRequest.bind(this);
+    }
+
+    componentDidMount(){
+        const daRef = firebase.database().ref('donating_agencies').child(this.props.account.agency);
+        daRef.on('value', (snapshot) => {
+            let donatingAgency = snapshot.val();
+            this.setState({
+                schoolID: donatingAgency.school
+            });
+        });
+    }
+
+    handleValidation(){
+        let fields = this.state.fields;
+        let errors = {};
+        let formIsValid = true;
+
+        //Date
+        if(!fields['startDate']){
+            formIsValid = false;
+            errors['startDate'] = 'Start date cannot be empty';
+        }
+
+        if(fields['endDate'] < fields['startDate']){
```

Comment:
are the values of type String or Date? You can compare like this? (If yes, that's cool!)

Reply:
They are type Date. I set fields in my handleChange function 
```
handleChange(field, e){         
        let fields = this.state.fields;
        fields[field] = e.target.value;        
        this.setState({fields});
}
```